packages/calypso-color-schemes/src/shared/color-schemes/_modern.scss

```diff
@@ -166,6 +166,7 @@ Uses custom theme highlight monochromatic palette for primary + accent
 	--color-sidebar-menu-hover-text: var(--theme-text-color);
 
 	/* Sidebar Hover - Nav unification */
+	--color-sidebar-submenu-selected-hover-text: var(--color-sidebar-submenu-hover-text);
```

Comment:
Let's move this below `--color-sidebar-submenu-hover-text`

Reply:
good idea. I'm applying this change to other schemes that will benefit of it as well.